charts/prompt-keda/values.yaml

```diff
@@ -113,6 +117,7 @@ alerts:
 
 registry:
   # -- If set, this application's Vault secret must contain a `central_repo_file` key containing a remote Butler configuration, and `instrument.centralRepo` is the local path where this file is mounted.
+  # If `instrument.readRepo` is also set, the secret must contain a `read_repo_file` key.
```

Comment:
Should this additional comment also be added to the 6 instrument-specific `prompt-keda-<instrument>/values.yaml`?